decko/lib/decko/config/environments/cucumber.rb

```diff
@@ -10,6 +10,7 @@
   # For more information see https://rspec.lighthouseapp.com/projects/16211/tickets/165
   config.cache_classes = true
 
+  config.persistent_cache = false
```

Comment:
upon further review, I think this is wrong.  it looks like cucumber was in fact using the persistent cache.  (https://github.com/decko-commons/decko/commit/75d4f685c05193fc6995a83801e1f5703a07d02e#diff-84503ec45e737516bfc7987cacc5839f)